ccpp/data/GFS_typedefs.F90

```diff
@@ -7139,8 +7136,7 @@ subroutine interstitial_phys_reset (Interstitial, Model)
     Interstitial%zngw            = clear_val
 
 ! CIRES UGWP v1
-    if (Model%do_ugwp_v1 .or. Model%gwd_opt==33 .or. Model%gwd_opt==22 &
```

Comment:
@mdtoy this PR reverts all changes that I had to make so that Evan could run the model w/o segfaults. Can you confirm that these changes are intended?